swarm/storage/netstore.go

```diff
@@ -57,7 +56,7 @@ func (self *NetStore) Get(key Key) (chunk *Chunk, err error) {
 			err := self.retrieve(chunk)
 			if err != nil {
 				// mark chunk request as failed so that we can retry it later
-				chunk.SetErrored(true)
+				chunk.SetErrored(ChunkErrTimeout)
```

Comment:
Just FYI - strictly speaking this is not necessarily `Timeout`, it could be any number of errors. I think if we have `no peers`, we get in this error condition too.

I don't know how to fix this better without refactoring the chunk, so this is not a suggestion to change it ;)

Reply:
`ChunkErrInaccessible`?

Reply:
Yes, better than timeout.